packages/ng/src/app/select/select.module.ts

```diff
@@ -1,9 +1,9 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { OverlayModule } from '@angular/cdk/overlay';
-import { LuSelectPicker } from './picker';
-import { LuSelectOption } from './option';
-import { LuSelectClearerComponent, LuSelectClearerFirstOrDefaultComponent } from './clearer';
+import { LuSelectPickerModule } from './picker';
+import { LuSelectOptionModule } from './option';
+import { LuSelectClearerModule } from './clearer';
 import { LuSelectDirective } from './directive';
```

Comment:
Why the directive isn't import in the same way as the others modules ?

Reply:
i didnt see select/sdirective had a module, i'll fix that asap